README.md

```diff
@@ -29,4 +29,37 @@ run `node driver.js` in command line
 
 ![uml](imgs/lab11.png)
 
-![uml](imgs/lab12.png)
\ No newline at end of file
+![uml](imgs/lab12.png)
+
+# LAB - CLASS 13
+
+## Project: Message Queues
+
+### Author: Mariko Alvarado
+
+### Links and Resources
+
+- [GHActions](https://github.com/Mariko-401-AdvancedJs/CAPS/actions)
+- [PR](https://github.com/Mariko-401-AdvancedJs/CAPS/pull/4)
+
+
+### Setup
+
+#### How to initialize/run your application (where applicable)
+
+run `node queue-server.js` in command line
+run `node flowers.js` in command line
+run `node widgets.js` in command line
+run `node driver.js` in command line
+
```

Comment:
This looks really lovely Mariko, the only thing that I would like to see going forward is if you require .env on root and in other folders, that you list on here what is needed and where